yatsm/cli/pixel.py

```diff
@@ -190,15 +191,27 @@ def pixel(ctx, config, px, py, band, plot, ylim, style, cmap,
         )
 
 
-def plot_TS(dates, y):
+def plot_TS(dates, y, seasons):
     """ Create a standard timeseries plot
 
     Args:
         dates (iterable): sequence of datetime
         y (np.ndarray): variable to plot
     """
     # Plot data
-    plt.scatter(dates, y, c='k', marker='o', edgecolors='none', s=35)
+    if seasons == []:
+        plt.scatter(dates, y, c='k', marker='o', edgecolors='none', s=35)
+    else:
+        for date in dates:
+            if date.month == 11 or date.month == 12 or date.month == 1 or date.month == 2 or date.month == 3:
```

Comment:
Could you define the dates up top somewhere? Would facilitate re-use in other parts of the script and/or altering the definitions via CLI.

``` python
# Placed up top somewhere
WINTER_MONTHS = (11, 12, 1, 2, 3)
...
```